### DISCUSSION TITLE
freyberg_pstfrom_pest_setup notebook issue

### DISCUSSION THREAD

**Discussion by Hydrok92** [gmdsi/GMDSI_notebooks#75] [Q&A] [answered]
Hey there,

I'm currently going through the freyberg_pstfrom_pest_setup notebook and ran into an error running:

pyemu.os_utils.run('pestpp-glm freyberg_mf6.pst', cwd=template_ws)

which results in the following error:

    124     if "window" in platform.platform().lower():
    125         if ret_val != 0:
    126             raise Exception("run() returned non-zero: {0}".format(ret_val))
    127     else:
    128         estat = os.WEXITSTATUS(ret_val)
    129     Exception: run() returned non-zero: 1

All previous notebooks have run fine, so i'm not sure if it's an issue on my side or not.  Any ideas would be much appreciated,

Cheers,

**Comment by jtwhite79** [comment]
Can you post a screenshot from the terminal so we can see what the pestpp-glm error was?  If you restart the kernel and run all, does the same error happen again?

**Reply by Hydrok92** [reply]
Hey there,

Sorry should have added that from the get go. Here it is! I guess it get's caught up on the forward_run.py script. 

<img width="607" alt="error_pstfrom" src="https://user-images.githubusercontent.com/82884061/218562204-2a136ffa-8719-4333-825e-a9a98102a9b1.PNG">

**Comment by jtwhite79** [comment]
looks like a pandas-pyemu version issue. can you check what version of pandas you are using?

**Reply by Hydrok92** [reply]
I've tried running the notebook with pandas versions 1.3.4, 1.4.4 and 1.5.0 without any luck

**Comment by mnfienen** [comment, marked as answer]
hi @Hydrok92 - can you confirm that your run is with Pandas 1.5.0?

This parameter changed in `to_csv` pretty recently
<img width="494" alt="image" src="https://user-images.githubusercontent.com/1110827/218792232-9b6e2fe5-7dd8-49bb-8e96-f994b22dd166.png">

To confirm, you may want to add a statement like `print(pd.__version__)` to your `forward_run.py` script to be sure that the version of pandas that is run by the forward model is the one you are expecting.

**Reply by Hydrok92** [reply]
Hey Mike, 

Strangely enough, when i downgraded pandas to 1.3.4 and went back up to 1.5.0 now it started working. Not sure if there was some weird package conflict going on somewhere. Thank you both for taking the time to help out!

**Reply by mnfienen** [reply]
Great! Glad that helped.
